docs/en/02_Developer_Guides/19_GraphQL/01_getting_started/02_configuring_your_schema.md

```diff
@@ -94,11 +98,11 @@ SilverStripe\GraphQL\Schema\Schema:
       src:
         - app/_graphql
         - module/_graphql
+        # The next line would map to an exposed folder e.g.: vendor/somevendor/somemodule/_graphql
+        - somevendor/somemodule: _graphql
```

Comment:
@unclecheese

I haven't tried this. I can give it a go if you are not 100% sure it works.